make/ex_doc_wrapper.erl

```diff
@@ -0,0 +1,54 @@
+-module(ex_doc_wrapper).
+
+-export([main/1]).
+
+main(Args) ->
```

Comment:
I tested this on OSX but I have no idea how it would behave on windows which I guess is somewhat concerning.

Reply:
CI passes for Windows so seems good!